openapi-core/src/main/java/io/ballerina/openapi/core/generators/service/ServiceDiagnosticMessages.java

```diff
@@ -47,7 +47,9 @@ public enum ServiceDiagnosticMessages {
             "Header '%s' with no header type can not be mapped to the Ballerina headers.",
             DiagnosticSeverity.ERROR),
     OAS_SERVICE_107("OAS_SERVICE_107", "HTTP status code '%s' is not supported in Ballerina.",
-            DiagnosticSeverity.ERROR);
+            DiagnosticSeverity.ERROR),
+    OAS_SERVICE_108("OAS_SERVICE_108", "Only array types of string, int, float, boolean, " +
+            "and decimal are allowed to be used in query parameters in Ballerina.", DiagnosticSeverity.ERROR);
```

Comment:
```suggestion
            "and decimal are allowed to be used as query parameters in Ballerina.", DiagnosticSeverity.ERROR);
```

Reply:
Resolved in https://github.com/ballerina-platform/openapi-tools/pull/1335/commits/445e23f5bd95629f18e244673178a695e7d65eda